<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>end</class>
 <widget class="QDialog" name="end">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 228, 16);</string>
   </property>
   <property name="text">
    <string>Winner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="no2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 50);
border:2px solid rgb(247, 228, 16);
border-radius:45px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>180</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 228, 16);</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 255, 16);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 228, 16);</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 228, 16);</string>
   </property>
   <property name="text">
    <string>NO.2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score1">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 255, 16);</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 228, 16);</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="no1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>220</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 254, 50);
border:2px solid rgb(247, 228, 16);
border-radius:45px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 228, 16);</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>550</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 255, 16);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>550</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(147, 97, 117, 255), stop:1 rgba(82, 54, 65, 255));
border-radius:15px;
border:1px solid rgb(132, 132, 132);
font:75 italic 18pt &quot;Bodoni MT&quot; ;
color:rgb(247, 255, 16);</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
